webpack-devserver.config.js

```diff
@@ -3,14 +3,7 @@ var config = require('./config')
 const webpack = require('webpack')
 
 var user = require('./demo/demo_user.json')
-var { createSignedUrl } = require('./server_utils/auth_utils')
-var {
-  acquireEmbedSession,
-  generateEmbedTokens,
-  setConfig,
-} = require('./server_utils/cookieless_utils')
-
-setConfig(config)
+var {addRoutes} = require('./server_utils/routes')
```

Comment:
seems prettier or the linter will complain here?

Reply:
js files are not linted or prettified. I've corrected manually.

Reply:
ah, right. sorry I missed the extension.